agent/runner.go

```diff
@@ -60,7 +60,7 @@ func NewRunner(workEngine rpc.Peer, f rpc.Filter, h string, state *State, backen
 	}
 }
 
-func (r *Runner) Run(ctx context.Context) error {
+func (r *Runner) Run(ctx context.Context, usePodman bool) error {
```

Comment:
instead of a pdman flag we should add a option struct

something like`type RunOptions struct {...}` in .../runner package

Reply:
Great will work on this during this feature

Reply:
@mfulz I merged master and adjusted the code a bit so it should now work with #463.